js/core/utils/position.js

```diff
@@ -34,7 +39,29 @@ const getBoundingRect = (element) => {
     return rect;
 };
 
+const getAvailableBoundsContainer = (container) => {
+    const $container = container && $(container);
+    let result;
+
+    if($container && $container.length && !isWindow($container.get(0))) {
+        const $containerWithParents = [].slice.call($container.parents());
+        $containerWithParents.unshift($container.get(0));
+
+        each($containerWithParents, function(i, parent) {
+            if(parent === $('body').get(0)) {
```

Comment:
Looks like we can just write condition as 
`
if(parent !== $('body').get(0) && window.getComputedStyle(parent).overflowY === 'hidden') {
    result = $(parent);
    return false;
}
`